demo/llm.rag.service/chat-serveragllmpluslb.yaml

```diff
@@ -46,6 +46,8 @@ spec:
             value: ${MODEL_TEMPERATURE}
           - name: RELEVANT_DOCS
             value: ${RELEVANT_DOCS}
+          - name: IS_JIRA_MODE
```

Comment:
Clarification: Could this be called "IS_JSON_MODE" because it looks like serveragllm is quite generic for JSON docs?

Reply:
sure, done